components/net/indexeddb/idb_thread.rs

```diff
@@ -304,17 +324,8 @@ impl IndexedDBManager {
                     origin,
                     name: db_name,
                 };
-                if self.databases.remove(&idb_description).is_none() {
-                    let _ = sender.send(Ok(()));
-                    return;
-                }
-
-                // FIXME:(rasviitanen) Possible security issue?
-                // FIXME:(arihant2math) using remove_dir_all with arbitrary input ...
-                let mut db_dir = self.idb_base_dir.clone();
-                db_dir.push(idb_description.as_path());
-                if std::fs::remove_dir_all(&db_dir).is_err() {
-                    let _ = sender.send(Err(()));
+                if let Some(db) = self.databases.remove(&idb_description) {
+                    db.delete_database(sender);
```

Comment:
Don't we want to send an `Ok(())` value in this branch as well? Do we need the `else` at all?